samples/client/petstore/c/Packing.cmake

```diff
@@ -0,0 +1,24 @@
+set(CPACK_PACKAGE_NAME lib${pkgName})
```

Comment:
@funzin is this here by accident?

Reply:
Reverted d1aa1c9

I excuted these commands and then the diff was appeared 🤔 
```
./mvnw clean package 
./bin/generate-samples.sh
```

Reply:
https://github.com/OpenAPITools/openapi-generator/runs/4324214455?check_suite_focus=tru
![Screen Shot 2021-11-25 at 22 37 26](https://user-images.githubusercontent.com/12893657/143451335-b652dac6-34dd-4625-907d-b347a5872918.png)
Maybe, these files shoud be tracked.